ReactWindows/ReactNative/DevSupport/DevServerHelper.cs

```diff
@@ -86,6 +88,25 @@ private bool IsHotModuleReplacementEnabled
         }
 
         /// <summary>
+        /// The Websocket proxy URL.
+        /// </summary>
+        public string WebsocketProxyUrl
+        {
+            get
+            {
+                return string.Format(CultureInfo.InvariantCulture, WebsocketProxyUrlFormat, DebugServerHost);
+            }
+        }
+
+        private string JavaScriptProxyHost
```

Comment:
It's a private property that will never become public.  

The only time I advocate for comments on non-public surfaces is when there is partial comment coverage of the public APIs on an internal class.  The reason I advocate for this is that it is sometimes the case that we need to refactor libraries and make internal classes public.  It's more of a policy of do everything, or nothing at all :-).

---
In reply to: [55510578](https://github.com/CatalystCode/react-native/pull/235#discussion_r55510578) [](ancestors = 55510578)